### ISSUE TITLE
PEX Hypercare Queries/Concerns

### ISSUE THREAD

**Issue by Sakthi1502** [beezlabs-org/l3-support#80]
Customer has raised a query regarding requests getting assigned to majority of users but couple of users are not getting assigned. This needs to be analysed based on the executions that happened this week and requests that got generated at the Docusign Portal. Once analysis is done, we will be able to determine whether this is an issue or not.

**Comment by Sakthi1502** [comment]
Update:
- The end user has another query regarding user still getting assigned to Docusign requests even when the respective user is removed from rule. Currently, there is a need to change in constants as well in Production to handle this scenario which will be kind of an recurring activity for support team and looking for an optimal way to overcome this. 
- Also, based on the analysis by support team, there is one user to whom requests are not getting assigned. To check on both these items, planning to connect with Developer.

**Comment by mbharathm** [comment]
@Sakthi1502 , as per my knowledge, there are no pending activities from our end for PEX. Can you let me know what is this activity about? If not please update the status correctly. Similarly on the parent task as well [#60](https://github.com/beezlabs-org/automation-dev/issues/60) PEX use case.

For the CR they have asked, please have a task in backlog and move it to Icebox for future consideration.

**Comment by Sakthi1502** [comment]
@mbharathm I will create a separate task for the CR/queries they have requested after hypercare closure. Whatever mentioned above are older issues and have been discussed with client as well. Closing this task since we had received an official hypercare closure on 21st December.
